docs/index.rst

```diff
@@ -3,8 +3,8 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to FireSim's documentation!
-===================================
+Welcome to FireSim's |version| documentation!
```

Comment:
```suggestion
Welcome to FireSim's documentation (version |version|)!
```

Will probably produce a more readable sentence, unless you're sure the version string will always be good.